downloads/archive/index.php

```diff
@@ -13,6 +13,8 @@
   require_once('./static-keys.php');
 
   // These variables should be progressively added if we update older versions.
+  // 14.0 onward uses 3 numbers instead of 4
+  $ver_windows_14 = "14.0.290";
```

Comment:
This is now 14.0.293 and will likely change again due to installation bug. issue #6619

Reply:
We can always make a new PR if necessary.